hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerDataScanner.java

```diff
@@ -49,6 +49,7 @@
   private final Canceler canceler;
   private final ContainerDataScrubberMetrics metrics;
   private final long dataScanInterval;
+  private static final String NAME = "ContainerDataScanner(";
```

Comment:
Can we use String format here?
```suggestion
  private static final String NAME_FORMAT = "ContainerDataScanner(%s)";
```